data/json/items/vehicle/engine.json

```diff
@@ -328,8 +329,9 @@
     "type": "ENGINE",
     "name": { "str": "6,000 HP gas turbine engine" },
     "description": "A massive gas turbine engine, used to power the V-22 Osprey.  Known for its high rate of fuel consumption.",
-    "weight": "900000 g",
-    "volume": "95 L",
+    "//": "Based on Rolls-Royce T406.",
+    "weight": "900 kg",
+    "volume": "4260 L",
```

Comment:
Most ground tiles can only contain up to 1000 L of items, so an item with a volume higher than 1000 L may disappear or make the game bug out (I didn't test which) when removed from a vehicle.

Reply:
Quite honestly these should probably be limited to 500L so that they can be carried in a cargo space otherwise the only way to transport them in a vehicle is to hold them in your lap while driving the car. Although it's less realistic that way as they don't totally accurately represent real volume there's presently no way to "share" cargo spaces for a single item. Being unrealistically small but still extremely large by the standards of most items is better than the alternative in terms of realism I think.

Reply:
@anoobindisguise while I do agree with you personally, in my experience the project owners seem to favor realism over convenience in almost every case and I don't think they would agree. @Maleclypse you've been following my pull requests, what do you think?

Reply:
The problem is that the game becomes less realistic by giving it a 4000L size. It becomes impossible to haul it anywhere. Unless you somehow hold it in your hands while driving the car. That's still possible with a smaller size but at least you can avoid that stupidity by putting it in a cargo space.

Reply:
I moved things above 1000L to 900L, lowered the i6 engine size to 500L, and undoubled the medium turbine volume. The only things that don't fit in a cargo space now are two of the turbine engines, which I think is reasonable. They're huge.